libnopegl/meson.build

```diff
@@ -278,7 +278,7 @@ hosts_cfg = {
 
 lib_deps = [
   cc.find_library('m', required: false),
-  dependency('libnopemd', version: '>= 11.0.0'),
+  dependency('libnopemd', version: '>= 11.0.1'),
```

Comment:
You can drop that we don't have a hard requirement.

Reply:
Ok.